src/analyse.ts

```diff
@@ -22,7 +23,7 @@ namespace Analyse {
 
     export async function analyse(
         manifestPath: string, analysisReportName: string,
-    ): Promise<"none" | "warning" | "error"> {
+    ): Promise<"none" | "warning" | "error" | ""> {
```

Comment:
it's a small thing, but I prefer `undefined` over the empty string here, it's more explicit